owner/id.go

```diff
@@ -80,6 +82,28 @@ func (id *ID) Parse(s string) error {
 	return nil
 }
 
+// Valid returns true iff id is a valid owner id.
```

Comment:
```suggestion
// Valid returns true if id is a valid owner id.
```

Reply:
Well it exists as a shortening for the conjunction https://wikiless.org/wiki/If_and_only_if?lang=en, but let's use more common language.

Reply:
wow, did not know about such shortening, thanks